server/build/docker/nginx/default.conf

```diff
@@ -24,7 +24,7 @@ server {
   }
 
   location / {
-    client_max_body_size 50M;
+    client_max_body_size 100M;
```

Comment:
Going one step further, should we perhaps consider updating [our docs](https://docs.mattermost.com/install/config-proxy-nginx.html)? I am thinking for consistency more than anything, considering our config default max file size [is actually 100MB](https://github.com/mattermost/mattermost/blob/ed58b10486a1325c473d31d7aeafdebe712f4d00/server/public/model/config.go#L1614) so we are literally recommending a potentially broken config.

Reply:
Makes sense, I'll create a PR for that if we have a consensus on 100mb :)